csa-runner-app/src/main/java/com/farao_community/farao/swe_csa/app/dichotomy/shifter/NetworkShifter.java

```diff
@@ -0,0 +1,22 @@
+package com.farao_community.farao.swe_csa.app.dichotomy.shifter;
+
+/*
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/**
+ * @author Jean-Pierre Arnould {@literal <jean-pierre.arnould at rte-france.com>}
+ */
+
+import com.farao_community.farao.dichotomy.api.exceptions.GlskLimitationException;
+import com.farao_community.farao.dichotomy.api.exceptions.ShiftingException;
+import com.farao_community.farao.swe_csa.app.dichotomy.variable.DichotomyVariable;
+import com.powsybl.iidm.network.Network;
+
+public interface NetworkShifter<U extends DichotomyVariable<U>> {
```

Comment:
use the SWE CC shifter (update it if needed) because they have extra stuff (like reconnecting generators & transformers)